packages/element-library/src/constants.ts

```diff
@@ -13,6 +13,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+// TODO (@AnuragVasanwala): Fix ESLint issue.
+export {
+  AUDIO_STICKERS,
+  AUDIO_STICKER_WIDTH_PRESETS,
+  AUDIO_STICKER_ASPECT_RATIOS,
+  AUDIO_STICKER_DEFAULT_PRESET,
+  AUDIO_STICKER_STYLES,
+} from './audio-sticker/constants';
```

Comment:
Are these only used in the editor? If so, let's define them in the `story-editor` package instead.